docs/managing-data-quality/snowplow-event-recovery/running/flink/index.md

```diff
@@ -16,15 +16,45 @@ sbt flink/assembly
 
 #### Running
 
-Using flink CLI:
+Event recovery jobs are usually ran using our [dataflow-runner](https://github.com/snowplow/dataflow-runner) application. A configuration template for running is [available in the repository](https://github.com/snowplow-incubator/snowplow-event-recovery/blob/master/.dataflow-runner).
+
+To run a transient EMR cluster and execute the job using the templates, download [Flink dataflow-runner templates](https://github.com/snowplow-incubator/snowplow-event-recovery/blob/master/.dataflow-runner) from the repository and run:
 
 ```bash
-flink run \
-  snowplow-event-recovery-flink-0.2.0.jar \
-  --input s3://bad-rows-location/** \
-  --output recovered-kinesis-topic \  
-  --failedOutput s3://unrecovered-collector-payloads-location/ \
-  --unrecoverableOutput s3://unrecoverable-collector-payloads-location/ \
-  --config $JOB_CONFIG \
-  --resolver $RESOLVER_CONFIG
-```
+dataflow-runner run-transient \
+  --emr-playbook flink-playbook.json.tmpl \
+  --emr-config flink-cluster.json.tmpl \
+  --vars bucket,$BUCKET_INPUT,region,$AWS_REGION,subnet,$AWS_SUBNET,role,$AWS_IAM_ROLE,keypair,$AWS_KEYPAIR,client,$JOB_OWNER,version,$RECOVERY_VERSION,config,$RECOVERY_CONFIG,resolver,$IGLU_RESOLVER,output,$KINESIS_OUTPUT,inputdir,$BUCKET_INPUT_DIRECTORY,interval,$INTERVAL
+  ```
+Where:
+- `BUCKET_INPUT` - S3 bucket containing bad events to be recovered (eg. geoffs-bad-events)
+- `BUCKET_INPUT_DIRECTORY` - directory in the `BUCKET_INPUT` to use as the source for bad events
+- `AWS_REGION` - region in which the job is being ran (eg. eu-central-1)
+- `AWS_SUBNET` - network subnet to run the job in (eg. subnet-435010347a21886ab)
+- `AWS_IAM_ROLE` - AWS IAM role to assume while running the job (eg. geoffs-recovery-role)
+- `AWS_KEYPAIR` - AWS EC2 key pair to use for the instances (eg. geoffs-keypair)
+- `JOB_OWNER` - tag to use to mark the owner of the job (eg. goeff)
+- `RECOVERY_VERSION` - application version (eg. 0.6.0)
+- `RECOVERY_CONFIG` - recovery job config as described in [Concepts](../concepts)
```

Comment:
```suggestion
- `RECOVERY_CONFIG` - recovery job config as described in [Concepts](../concepts/index.md)
```
Here and below — see README for the docs repo on why this is needed

Reply:
@stanch I believe this is what I had before, but this fails the build, so I turned to top-level link.

Reply:
It fails the build because the link is wrong :) So actually the link checking feature works as expected!
For some reason, there is no `../concepts/index.md`, but there is `../0-2-concepts/index.md`.

Reply:
https://docs.snowplow.io/docs/managing-data-quality/snowplow-event-recovery/concepts/ — this is a 404

Reply:
Alternatively, feel free to rename that dir to `concepts` and add a redirect in `static/_redirects`